### ISSUE TITLE
ML-Benchmark prediction submission: [DeepESD]

### ISSUE THREAD

**Issue by rtomeidl** [WCRP-CORDEX/ml-benchmark#19]
### Institution ID

IDL

### Institution Name

Instituto Dom Luiz, Faculdade de Ciências da Universidade de Lisboa

### Emulator identifier (`emulator_id`)

DeepESD_IDL_v1

### Emulator Description

Deep convolutaional neural network that includes 3 convulation layers and ond dense layer, with relu activation functions. 
Input data is standardized at gridbox level using mean/std in the training period.
For precipitation a bergamma loss function is used and for maximum temperature the standart mse.

### Hardware and Training Details

One node with one NVIDIA A100 GPU (40GB).
Training time from 10 to 30 minutes depending on variable/emulation method (maximum of 10000 epochs).
Inference time roughly 2 hour for the entire db (On one node without GPU).

### Stochastic/Probabilistic Output

no

### Reference URL

https://doi.org/10.5194/gmd-17-229-2024

### Repository for Reproducibility

https://github.com/rtomeidl/CordexML_DeepESD

**Comment by jgonzalezab** [comment]
HI @rtomeidl. I went through your submission and it looks like the dates are off for most of the data files (e.g. files that should cover 2041–2060 are covering 1961–1980 instead) Could you check and resubmit? When computing predictions, make sure to stick to the period of each corresponding set of predictors. Upload the fixed version as `DeepESD_IDL_v1_v2.zip`

Thanks!

**Comment by IDLGroupRG1** [comment]
Hey! 
Sorry the files are fine but i did mess up the units on the time dimension. I already correct the files and uploaded for both submissions.

**Comment by jgonzalezab** [comment]
Thanks for the fix @rtomeidl. Unfortunately there is still a file with the wrong dates (off by just a couple of years). Would you mind fixing it? I only need you to send this files (maybe in a zip file named something like <emulator_name>_fixed_file.zip). Thanks!!

```
DeepESD_IDL_v2/SA_Domain/Emulator_hist_future/mid_century/imperfect/Predictions_pr_tasmax_NorESM2-MM_2041-2060.nc
```

**Comment by IDLGroupRG1** [comment]
Hey! Uploaded v3... sorry found a error in the inference. Hopefully the dates are fine now!
